HW04/aleksei_zhuchkov/spec/mentor_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require_relative '../lib/mentor'
+require_relative '../lib/notification'
+require_relative '../lib/student'
+require_relative '../lib/homework'
+
+RSpec.describe Mentor do
+  let(:student) { Student.new(name: 'student_name', surname: 'student_surname') }
+  let(:mentor) { described_class.new(name: 'mentor_name', surname: 'mentor_surname') }
+  let(:homework) do
+    mentor.add_homework(title: 'test_title',
+                        description: 'test_description',
+                        student: student)
+  end
+
+  describe 'check method add_homework' do
+    it 'add homework to student to be a instance of class Homework' do
```

Comment:
for instance,
 ```suggestion
    it 'returns the homework of Homework class ' do
```

Reply:
I have done it
https://github.com/labs-ruby/homeworks-2022/pull/122/commits/e8a5c1a9d8255e65ba03626bca1c05d49769af10